devices/AtModem/DTOs/BatteryStatus.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using UnitsNet;
+
+namespace IoT.Device.AtModem.DTOs
+{
+    /// <summary>
+    /// Represents the battery status.
+    /// </summary>
+    public class BatteryStatus
+    {
+        /// <summary>
+        /// Gets the battery charge status.
+        /// </summary>
+        public BatteryChargeStatus Status { get; }
+
+        /// <summary>
+        /// Gets the charge level of the battery.
+        /// </summary>
+        public Ratio ChargeLevel { get; }
```

Comment:
Do we need to depend & ref 2 UnitsNet libraries _just_ to format the charge ration and battery voltage?

Reply:
Yes, that's our rule all up for all bindings. I removed the ratio from the signal class, as it's not really a ratio. I left it with the battery.